localization/french/artifacts/wc_artifacts_l_french.yml

```
@@ -0,0 +1,24 @@
+﻿l_french:
+
+ terenas_crown_name:0 "Couronne de Lordaeron"
+ terenas_crown_desc:0 "Une couronne à pointes utilisée pour le couronnement des hauts rois de Lordaeron."
+ frostmourne_name:0 "Deuillegivre"
+ frostmourne_desc:1 "Un [inventory_artifact|E], une lame runique maudite de [GetTrait('lich_king').GetName( GetNullCharacter )]. La lame emprisonne les âmes, augmentant la $MOD_PROWESS$ de la lame et $MOD_DREAD_BASELINE_ADD$ #weak (jusqu'à $MOD_PROWESS$ et 30 $MOD_DREAD_BASELINE_ADD$)#!. $frostmourne_name$ ne perd pas de [durability|E]. Le porteur transforme la victime [GetTrait('being_undead').GetName( GetNullCharacter )]. Dans un tel cas, l'âme du mort est pervertie : il obtient un [traits|E] mauvais comme [GetTrait('callous').GetName( GetNullCharacter )], [GetTrait('vengeful').GetName( GetNullCharacter )], etc."
+ gorehowl_name:0 "Hurlesang"
+ gorehowl_desc:0 "Une hache massive brandie par Grommash Hurlenfer du clan Warsong. Elle est marquée de façon permanente du sang de ses ennemis tombés au combat. Les petits trous et encoches qui parsèment la lame la font « chanter » lorsqu'elle est brandie."
+
+ hammer_of_twilight_name:0 "Marteau du crépuscule"
+ hammer_of_twilight_desc:0 "Forgée avec de l'élémentium, de la magie du Vide et de la folie, cette arme est un symbole de malveillance et de puissance du culte du Marteau du Crépuscule et de ses maîtres des Dieux très anciens."
+ 
+ dark_crystal_name:0 "Cristal Noir"
+ dark_crystal_desc:0 "La magie du Vide émane de cet étrange cristal, mais quelque part dans ses profondeurs, vous ressentez la présence de la Lumière."
+
+ ### Draenei
+ naaru_staff.name:0 "Bâton du rédempteur"
+ naaru_staff.desc:0 "Un bâton long et mince fabriqué à partir de lumière pure, offert par les Naaru à Velen, le chef des Draeneï. Il sert de symbole d’espoir et de foi tout en étant également un puissant outil d’orientation et de protection."
```

```suggestion
 naaru_staff.desc:0 "Un bâton long et mince fabriqué à partir de lumière pure, offert par les Naaru à Velen, le chef des Draeneï. Il sert de symbole d’espoir et de foi tout en étant également un puissant outil de guidance et de protection."
```
Ça ne se dit pas vraiment orientation dans ce contexte.